mriqc/__about__.py

```diff
@@ -71,7 +71,7 @@
     'nipy',
     'statsmodels',
     'versioneer',
-    'xvfbwrapper',
+    'xvfbwrapper'
```

Comment:
Please add the trailing comma so this file does not show differences in two lines when adding dependencies.